packages/venia-concept/src/components/MiniCart/product.js

```diff
@@ -15,19 +15,26 @@ class Product extends Component {
     static propTypes = {
         classes: shape({
             image: string,
+            modal: string,
             name: string,
             optionLabel: string,
-            optionValue: string,
             options: string,
             price: string,
             quantity: string,
             quantityOperator: string,
+            quantityRow: string,
             quantitySelect: string,
             root: string
         }),
         item: shape({
             item_id: number.isRequired,
             name: string.isRequired,
+            options: arrayOf(
+                shape({
+                    label: string,
+                    value: string
+                })
+            ),
```

Comment:
Since these options will be present on the `item` because we merged them from `totals` in the data-layer, you can remove `totalsItems` from `product.js` entirely.